action/protocol/staking/contractstaking/dirty_cache.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2023 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package contractstaking
+
+import (
+	"math/big"
+	"sync"
+
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/db/batch"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+)
+
+const (
+	// bucket related namespace in db
+	_StakingBucketInfoNS = "sbi"
+	_StakingBucketTypeNS = "sbt"
+	_StakingNS           = "s"
```

Comment:
use a 2-letter namespace to be consistent

Reply:
ok